apps/api/src/services/queue-worker.ts

```diff
@@ -192,6 +192,11 @@ async function processJob(job: Job, token: string) {
       job,
       token,
     });
+
+    // Better if we throw here so we capture with the correct error
+    if(!success) {
```

Comment:
@mogery added this here. because when success was false we were just continueing, throwing type errors when it was actually other errors in runWebScraper. Is this okay?